core/services/functions/listener.go

```diff
@@ -19,7 +19,7 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/generated/ocr2dr_oracle"
 	"github.com/smartcontractkit/chainlink/v2/core/logger"
 	"github.com/smartcontractkit/chainlink/v2/core/services/job"
-	"github.com/smartcontractkit/chainlink/v2/core/services/ocr2/plugins/functions/config"
+	config "github.com/smartcontractkit/chainlink/v2/core/services/ocr2/plugins/functions/config"
```

Comment:
I don't think you need any of this any more.

Reply:
I do need it. I tried to remove.

Reply:
Do you mean the whole import, or the alias?
```suggestion
	"github.com/smartcontractkit/chainlink/v2/core/services/ocr2/plugins/functions/config"
```
Aliasing `config` as `config` has no effect and is usually removed by gofmt/goimports or IDEs.